Assets/Scripts/Editor/BuildHelpers.cs

```diff
@@ -0,0 +1,235 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+
+/// <summary>
+/// Utility menus to easily create our builds for our playtests. If you're just exploring this project, you shouldn't need those. They are mostly to make
+/// multiplatform build creation easier and is meant for internal usage.
+/// </summary>
+internal static class BuildHelpers
```

Comment:
Should we wrap this functionality behind a scripting symbols? Otherwise that "feature" will be released to the public and I am not sure if we want to support it

Reply:
So this is in the Editor folder, in boss room's project (not in our Utilities package). Editor folders are already excluded from builds. 
We also don't really have the concept of "APIs to support" in samples, at least not the same way Packages do. We write code examples that users study and copy to their own games.
If it was in our Utilities package, I would have agreed though, this one is closer to other packages.